src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,95 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final double DEFAULT_LOAD_FACTOR = 0.75d;
+    private static final int GROW_NUMBER = 2;
+    private Node<K, V>[] table;
+    private int threshold;
```

Comment:
`threshold` in redundant variable, you can simple use table.length * LOAD_FACTOR where it is needed